stacks-signer/src/client/stacks_client.rs

```diff
@@ -591,4 +626,26 @@ mod tests {
         );
         assert!(h.join().unwrap().is_ok());
     }
+
+    #[test]
+    fn core_info_call_for_consensus_hash_should_succeed() {
+        let config = TestConfig::new();
+        let h = spawn(move || config.client.get_stacks_tip_consensus_hash());
+        write_response(
+            config.mock_server,
+            b"HTTP/1.1 200 OK\n\n{\"stacks_tip_consensus_hash\": \"4e99f99bc4a05437abb8c7d0c306618f45b203196498e2ebe287f10497124958\"}",
```

Comment:
The consensus hash is a 20-byte hash, not a 32-byte hash.

Reply:
Fixed.